app/scripts/modules/kubernetes/src/v2/serverGroup/details/details.controller.ts

```diff
@@ -161,8 +163,26 @@ class KubernetesServerGroupDetailsController implements IController {
       this.serverGroup = this.transformServerGroup(serverGroupDetails);
       this.serverGroup.account = fromParams.accountId;
       this.state.loading = false;
+      this.serverGroup.entityTags = this.extractEntityTags();
+      this.configureEntityTagTargets();
     });
   }
+
+  private extractEntityTags(): IEntityTags {
```

Comment:
I'm curious why this doesn't need to happen for load balancers and security groups

Reply:
good question

Reply:
loadbalancer is extracted from the app data. security group makes an api call then another api call for entity tags. for google, servergroup makes an api call then gets entity tags from app data, so I did that.